reference/sectionA_definitions.inc

```diff
@@ -4,8 +4,8 @@
 
 **Authors:**
 
-Michael T. Cooling\*
 Michael Clerx
```

Comment:
I like this of course, but should be not have a chat about first/last authors at some stage?

Reply:
I just changed the order to be alphabetic as he was no longer the corresponding author ... discussion welcome though :)

Reply:
I agree with an alphabetical order, but using the first name. Thanks. 😉

More seriously, an alphabetical order using the last name is what I would expect here.